Change the call + FAQs (#595)
ropensci/roweb3: content/champions/programdetailsmentors_2023.md

Author: Yes!

+++
title = "Program Details - Mentors"
description = "Mentors support our cohort of rOpenSci Champions. Could you offer insight and advice to our selected Champions? Learn the details and express your interest!"
tags = ["champions"]
+++

{{< figure src = "/images/volunteer/peep1.png" width = "300" alt = "A person with his dog working on a notebook." class = "pull-left" >}}

The rOpenSci Champions Program is for _people from historically and systematically excluded groups_ who are interested in contributing to rOpenSci and the broader ecosystem of open source and open science communities. After receiving training, mentors will work with Champions to develop their project, which can be one of the following options:

* Create a new package;

* Go through the review process with an R package they have already developed;

* Serve as a reviewer for the rOpenSci software peer review system.

Mentors may also support Champions with delivering other activities including: giving a talk, writing blog posts, and teaching other community members.

### Role description

> Mentors will play a vital role in the rOpenSci Champions Program by helping select the [first cohort of Champions](/blog/2022/09/22/launch-champions-program/#how-to-apply) and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, and organizations; __recommend__, resources, readings, training, and experiences; and provide __feedback__ on the mentee's project.

Reviewer: ```suggestion
Mentors will play a vital role in the rOpenSci Champions Program by helping select [Champions](/blog/2023/07/14/call-champions-program/) and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, and organizations; __recommend__, resources, readings, training, and experiences; and provide __feedback__ on the mentee's project.
```

We are seeking mentors from the rOpenSci community who have:

* Demonstrated an understanding and ability to work openly and collaboratively;

* The willingness to guide and teach others;

* Previous experience with any of the following: 

  * Developing, maintaining or contributing to any rOpenSci package or package which has gone through rOpenSci review, or
  
  * Participating in the rOpenSci review process, or 
  
  * Package development and/or peer code review.

> Mentors will gain mentoring experience and skills, be acknowledged on the rOpenSci website and social media channels, and make new connections across the Champions' networks. By mentoring our selected Champions and helping to shape this program by providing feedback on this pilot year, mentors will also be giving back to the rOpenSci community in a great and meaningful way!

Reviewer: ```suggestion
Mentors will gain mentoring experience and skills, be acknowledged on the rOpenSci website and social media channels, and make new connections across the Champions' networks. By mentoring our selected Champions and helping to shape this program by providing feedback, mentors will also be giving back to the rOpenSci community in a great and meaningful way!
```

### Expectations

> The total time commitment is expected to be approximately 1-2 hours per month from October 2023 - October 2024 to complete the following tentative program timeline.

Reviewer: Update date/times
```suggestion
The total time commitment is expected to be approximately 1-2 hours per month from October 2023 - October 2024 to complete the following tentative program timeline.
```

Author: That is the correct timeline for this call.

Reviewer: Oh good! I think I was confused about when it started (clearly!)

|Date|Event|Detail|Duration|
|----|-----|------|--------|
|October, 2023|Select Champions |Review, rank and select 1-2 reviewed projects to mentor.|2 h|
|November 2023|Kick-off event|Learn about the program, review the program expectations, and address questions. Meet mentees.|1.5 h|

> |January - February 2023|Training|How to mentor and logistics on the program.|1 h|

Author: ```suggestion
|January - February 2024|Training|How to mentor and logistics on the program.|1 h|
```

|March 2023|Cohort meeting: Presentation of projects|Champions give 5 minutes presentations on their project (package review or development) and get feedback.|1.5 h|
|March-July 2023|Meeting with mentee|Provide feedback and guidance on mentee project.  Connect with other experts, mentors or community members. Recommend training, resources for the mentee project.|1 meeting per month 0.5 h|
|Jun 2023|Cohort meeting: Reflections on projects|Champions will be invited to share takeaways from their projects with the cohort.|1.5 h|
|August - November 2023|Meeting with mentee|Provide feedback and guidance on mentee project.  Connect with other experts, mentors or community members. Recommend training, resources for the mentee project. |1 meeting per month 0.5 h|
|September 2023|Cohort meeting: Reflections on events|Champions will be invited to share takeaways from their events with the cohort.|1.5 h|

> |December 2023|Cohort meeting: Reflections on program|Mentors will be invited to share takeaways from the program and help to define future roles for Champions.|1.5 h|

Reviewer: Update this timetable

|December 2023|Cohort meeting: Reflections on program|Mentors will be invited to share takeaways from the program and help to define future roles for Champions.|1.5 h|

{{< figure src = "/images/volunteer/peep2.png" width = "300" alt = "A person with his dog working on a notebook." class = "pull-right" >}}

### Benefits of being a mentor

By becoming an rOpenSci mentor you will:

* Empower members of historically and systematically excluded groups by sharing your expertise with others; 
* Connect with other mentors and with the champion community, expanding your professional network;
* Gain mentoring experience and skills;
* Gain exposure, recognition and acknowledgment for your work as a mentor in rOpenSci spaces (website, social media, events)
* Help rOpenSci and the wider R community grow;
* Co-develop this program and other rOpenSci projects by sharing feedback;

Mentors will participate in a training about mentoring and program logistics as part of their onboarding process and have the support of the Community Manager throughout the program.

> ### How to apply

Author: ```suggestion
```
